packages/hydrogen/src/sitemap/sitemap.ts

```diff
@@ -9,6 +9,8 @@ const SITEMAP_PREFIX = `<?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">`;
 const SITEMAP_SUFFIX = `</urlset>`;
 
+const SITEMAP_STOREFRONT_API_VERSION = '2024-10';
```

Comment:
Since this is going out with the 2024-10 release, as I understand it we will no longer need to provide the `storefrontApiVersion` override to the `storefront.query` call, so I'll remove this `const` and rebase this against #2570